<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>spectrumVisualization</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="spectrumLayout_vbox"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="loadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load spectrum on fitLine</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
